src/BladeCommentsPrecompiler.php

```diff
@@ -39,4 +39,25 @@ protected static function commenters(): array
             ->map(fn (string $class) => app($class))
             ->toArray();
     }
+
+    /**
+     * Insert our comment prefix
+     *
+     * @param  null|string  $string
+     */
+    public static function insertPrefix($string): ?string
```

Comment:
Instead of using a separate function, could you implement the prefix in the individual commenters?

Reply:
But that would make changing it harder right? What is your concern with this function?

Reply:
An alternative I can think of is for all commenters to inherit from a base class, but there are request and blade commenters, so I think it will still need 2 base classes

Reply:
> But that would make changing it harder right? What is your concern with this function?

I feel that just adding it in the commenters results in simpler code. The slightly elevated maintenance burden is an acceptable tradeoff for simpler code in this case.